auth-web/src/components/auth/account-settings/team-management/InvitationsDataTable.vue

```diff
@@ -97,6 +100,10 @@ export default class InvitationsDataTable extends Vue {
     }
   ]
 
+  private canApproveOrDeny (): boolean {
+    return !this.currentUser.roles?.includes(Role.ContactCentreStaff)
```

Comment:
Does this mean it will show the approve / deny buttons for a normal USER? has this been tested?